ci/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:22.04 AS build-env
+
+ARG LIB_ROCKS_SYS_VER
+
+RUN apt-get update -yy && \
```

Comment:
Remove the duplicated 'y'

Reply:
`-yy` seems to be the abbreviation of `--yes --force-yes`, which the later one could omit any configuration.